src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
+
+interface PaddingProps {
+  sizeXS?: SpacingUnit | 0
+  size?: SpacingUnit | 0
+}
+
+const responsivePadding = media.sm<PaddingProps>`
+  padding-right: ${p => space(p.sizeXS as SpacingUnit) || 0}px;
+  padding-left: ${p => space(p.sizeXS as SpacingUnit) || 0}px;
+`
+
+export const HorizontalPadding = styled.div.attrs<PaddingProps>({})`
+  padding-right: ${p => space(p.size as SpacingUnit) || 0}px;
+  padding-left: ${p => space(p.size as SpacingUnit) || 0}px;
+  margin-right: auto;
+  margin-left: auto;
+  ${responsivePadding};
+`
+
+HorizontalPadding.defaultProps = {
```

Comment:
These defaults are equivalent to the padding previously applied in `Boot`. We could have each component pass them in as props instead to avoid setting defaults.